### PR TITLE
Added Error message for KIA0401

### PR DESCRIPTION
**Pull request by hhovsepy**
Related to https://github.com/kiali/kiali/pull/5693

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseURL = "https://staging.kiali.io"
+baseURL = "https://kiali.io"
```
Comment (jshaughn) [inline config.toml:1]: hmm, I guess this is correct, not sure why it was set to staging.kiali.io.

File: content/en/docs/Features/validations.md [hunk 2]
```diff
@@ -891,6 +891,32 @@ Add missing Service Entry which address will match the Workload Entry's address.
 - [Validator source code](https://github.com/kiali/kiali/tree/v1.52.0/business/checkers/serviceentries/workload_entry_address_match.go)
 
 
+## K8s HTTPRoutes {#httproutes}
+
+### KIA1401 - HTTPRoute is pointing to a non-existent K8s gateway
+
+Gateway API HTTPRoute could be pointing to a Gateway (K8s) the Route wants to be attached to. When the namespace field it not specified, it takes Gateways from currect HTTPRoute's namespace. Here the error indicates that the pointed Gateway is not found in the provided namespace.
```
Comment (jshaughn) [inline content/en/docs/Features/validations.md]:
```suggestion
Gateway API HTTPRoute could be pointing to a [k8s] Gateway that the Route wants to be attached to. When the namespace field is not specified it takes Gateways from the current HTTPRoute's namespace. Here the error indicates that the referenced Gateway is not found in the provided namespace.
```

**Review by jshaughn — CHANGES_REQUESTED** [review]

File: config.toml (hunk 1 above)
Comment (hhovsepy) [reply config.toml:1]: It seems I did a wrong rebase.

**Review by jshaughn — APPROVED** [review]
LG!  Does this also need to go into Staging?  If only merged to Current it will not survive after the next release..

**Comment by hhovsepy** [comment]
Then isn't it better to send into Staging only?

**Comment by jshaughn** [comment]
In general changes go to `staging` because then it will show up on the next release and will live on.  `staging` is basically "master" for kiali.io.   `current` is the most recent release (so in this case it reflects v1.62).  Sometimes we also want to backport to `current` if the change is relevant to code that is already released.  So yes, in general you want `staging`, sometimes also `current`, but very rarely only `current`.  If `KIA0401` is relevant to our released version, v1.62, then yes we want a PR for `current` but  Probably you want to re-target this to `staging`, or have an addition PR for `staging`.

**Comment by hhovsepy** [comment]
Thank you Jay, not it is clear. Closing this PR in favor of https://github.com/kiali/kiali.io/pull/617.

**Pull request closed without merging**